docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -169,218 +168,267 @@ Example
+=======================
 
-After fitting, the ``RidgeReducer.transform`` method can be used to generate *X1* from *X0*.
+After fitting, the ``RidgeReducer.transform`` method can be used to generate :math:`X^1` from :math:`X^0`.
 
 Parameters
 ----------
 
-- ``block_df``: Spark DataFrame representing the beginning block matrix.
-- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models.
-- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs.
-- ``model_df``: Spark DataFrame produced by the RidgeReducer fit method, representing the reducer model.
+- ``block_df``: Spark DataFrame representing the beginning block matrix
+- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models
+- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs
+- ``model_df``: Spark DataFrame produced by the ``RidgeReducer.fit`` function, representing the reducer model
 - ``covariates``: Pandas DataFrame containing covariates to be included in every model in the stacking
   ensemble (optional).
 
 Return
 ------
 
-The output of the transformation is closely analogous to the block matrix DataFrame we started with.  The main
-difference is that, rather than representing a single block matrix, it really represents multiple block matrices, with
-one such matrix per label (phenotype).  Comparing the schema of this block matrix DataFrame (``reduced_block_df``) with
-the DataFrame we started with (``block_df``), the new columns are:
+The output of the transformation is closely analogous to the block matrix DataFrame we started with.  The main difference is that, rather than representing a single block matrix, it really represents multiple block matrices, with one such matrix per label (phenotype).  Comparing the schema of this block matrix DataFrame (``reduced_block_df``) with the DataFrame we started with (``block_df``), the new columns are:
 
-- ``alpha``: This is the name of the alpha value used in fitting the model that produced the values in this row.
-- ``label``: This is the label corresponding to the values in this row.  Since the genotype block matrix *X0* is
-  phenotype-agnostic, the rows in ``block_df`` were not restricted to any label/phenotype, but the level 1 block
-  matrix *X1* represents ridge model predictions for the labels the reducer was fit with, so each row is associated with
-  a specific label.
+- ``alpha``: Name of the alpha value used in fitting the model that produced the values in this row
+- ``label``: The label corresponding to the values in this row.  Since the genotype block matrix :math:`X^0` is phenotype-agnostic, the rows in ``block_df`` were not restricted to any label (phenotype), but the level 1 block matrix :math:`X^1` represents ridge model predictions for the labels the reducer was fit with, so each row is associated with a specific label.
```

Comment:
It would be helpful to paste in the output of `df.printSchema()` so that readers don't need to refer back to the previous schema.

Reply:
Added.